tests/vcd_disk.py

```diff
@@ -180,5 +180,16 @@ def test_010_get_storage_profile(self):
         print("Storage Profile: " + str(profile))
         assert profile.get('name') == self.config['vcd']['storage_profile']
 
+    def test_111_change_owner_disk(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, resource=logged_in_org)
```

Comment:
missing space around =

Reply:
Just followed the convention from other methods. No space.